sciplot/PlotBase.hpp

```diff
@@ -96,8 +96,13 @@ class PlotBase
     /// Draw plot object with given `what`, `using` and `with` expressions (e.g., `plot.draw("sin(x)*cos(x)", "", "linespoints")`,  (e.g., `plot.draw("file.dat", "1:2", "points")`)).
     auto draw(const std::string& what, const std::string& use, const std::string& with) -> DrawSpecs&;
 
+    /// Draw plot object using completely custom plot commands (e.g., `plot.draw('file.dat' using 1:2 with linespoints pointtype 5 linecolor rgb variable`)`.
```

Comment:
```
/// Draw plot object using completely custom plot commands (e.g., `plot.draw('file.dat' using 1:2 with linespoints pointtype 5 linecolor rgb variable`)`.
    /// @note This method only has an effect if custom commands are enabled using useCustomDrawCmds(true).
    /// @note It can only be combined with other custom draw commands, not with normal ones.
```